app/services/gitlab_integration/client_wrapper.rb

```diff
@@ -22,18 +22,53 @@ def add_deployqa_bot_to_repo(repository, permission: :guest)
       gitlab_client.add_team_member(repository.integration_id, ENV["GITLAB_DEPLOYQA_BOT_ID"], permission_code)
     end
 
```

Comment:
Please try to separate to `bot` and `user` - they have different use-cases and methods. Also please consider move it to lib dir into `clients` namespace